smartsim/_core/mli/message_handler.py

```diff
@@ -126,7 +129,9 @@ def build_model(data: bytes, name: str, version: str) -> model_capnp.Model:
         return model
 
     @staticmethod
-    def build_model_key(key: str) -> data_references_capnp.ModelKey:
+    def build_model_key(
```

Comment:
need to update doc string for ``feature_store_descriptor``

Reply:
fixed